.github/workflows/publish.yaml

```diff
@@ -1,16 +1,13 @@
 name: Publish
 
-on: workflow_dispatch
+on:
```

Comment:
So this is triggered when a GitHub package is published? I usually do things in the opposite order: publish to PyPI then publish to GitHub. I see the GitHub packages as just a nice to have but useless right now. In fact what I would prefer is still triggering on `workflow_dispatch` but having the action upload to both GitHub packages and PyPI.

Reply:
No, 
```
on:
  release:
```
means that it will trigger when you're creating a tag with description == release. Here are the [docs](https://github.com/AustinScola/pytest-motor/releases).